Food_Tracker_App/app/src/main/java/com/example/foodtracker/Ingredient.java

```diff
@@ -0,0 +1,67 @@
+package com.example.foodtracker;
+
+public class Ingredient {
+    private String description;
+    private int quantity;
+    private String location;
+    private String category;
+    private int amount;
+    private String expiry;
+
+    public Ingredient(String description, int quantity, String location, String category, int amount, String expiry){
+        this.description = description;
+        this.quantity = quantity;
+        this.location = location;
+        this.category = category;
+        this.amount = amount;
+        this.expiry = expiry;
```

Comment:
Use the setters here, since we may want to add some restrictions on setting fields